setup.py

```diff
@@ -24,16 +24,14 @@
 ##
 """
 This script can be used to install easybuild-framework, e.g. using:
-  easy_install --user .
-or
-  python setup.py --prefix=$HOME/easybuild
+  python setup.py install --prefix=$HOME/easybuild
 
 @author: Kenneth Hoste (Ghent University)
 """
 import glob
 import os
-from distutils import log
-from distutils.core import setup
+import logging
+from setuptools import setup
```

Comment:
Wouldn't it make sense to keep using `setup` from `distutils` when it's still available, and only use `setuptools` as a fallback?

```suggestion
try:
    from distutils.core import setup
except ImportError:
    from setuptools import setup
```

Reply:
sure, although I think setuptools is only necessary on the packaging side, if you simply do `pip install easybuild` you don't need it. I'll apply it anyway since it avoids needing (modern?) setuptools for people stuck with older pythons.